src/Anthropic.Client.Tests/Services/Messages/MessageServiceTest.cs

```diff
@@ -38,6 +42,213 @@ public async Task CreateStreaming_Works()
         }
     }
 
+    [Fact]
+    public async Task CreateStreamingAggregation_WorksNoContent_RawMessageStartEvent()
+    {
+        // arrange
+        var messagesServiceMock = new Mock<IMessageService>();
+        static async IAsyncEnumerable<RawMessageStreamEvent> GetTestValues()
+        {
+            yield return new(new RawMessageStartEvent(GenerateStartMessage()));
+            yield return new(new RawMessageStopEvent());
+            await Task.CompletedTask;

```

Comment:
What's this for? Is it for suppressing some lint around `async` methods without any `await`?